smart-contracts/.env.example

```diff
@@ -2,23 +2,26 @@
 #    General
 # ------------
 CONSENSUS_THRESHOLD=100
+# This is the address of the owner of the upgradeable proxy
 ETHEREUM_PRIVATE_KEY="c87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3"
+
 # ------------
 #    Local
 # ------------
 LOCAL_PROVIDER="http://localhost:7545"
-# Local validator count must be at least 1 and no more than 8 
 
 # ------------
 #   Network
 # ------------
 # Owner of the bridgebank, this must be set when we are deploying to tesnet or mainnet
 OWNER='0x627306090abaB3A6e1400e9345bC60c78a8BEf57'
+# Address of the pauser
+PAUSER='0x627306090abaB3A6e1400e9345bC60c78a8BEf57'
```

Comment:
Any chance we could change that environment variable name to something more specific like ETHEREUM_PAUSER? Having very generic environment variables like OWNER and PAUSER in setups that have to deal with both blockchains is confusing.